app.js

```diff
@@ -2,87 +2,747 @@
 
 // List 3 expressions that surprised you or gave unexpected results, why?
 
-// List 2 exprsssions you felt comfortable interpreting at a glance.
+2 + 'pizza' // I thought it would throw an error because 2 is not a string
+5 === 5.0 // With strict equality I thought it might be false, but I realize those numbers are identical
```

Comment:
JS actually sees numbers as floats so if you input `5` JS sees it as `5.0` which is why these are equal